src/constants/settings.ts

```diff
@@ -0,0 +1,84 @@
+import {ComponentType, SVGProps} from "react";
+import {GeneralSettings, Participants, SettingsAppearance, SettingsFeedback, Share, FileDefault} from "components/Icon";
+import {Color} from "./colors";
+
+export const MOBILE_BREAKPOINT = 920;
+
+export type MenuKey = "board" | "participants" | "appearance" | "share" | "export" | "feedback" | "profile";
+
+type LocalizationKey = "BoardSettings" | "Participants" | "Appearance" | "ShareSession" | "ExportBoard" | "Feedback" | "Profile";
+
+export type MenuItem = {
+  localizationKey: LocalizationKey;
+  location: string;
+  isModeratorOnly: boolean;
+  color: Color;
+  icon: ComponentType<SVGProps<SVGSVGElement>> | "profile"; // profile: special case
+};
+
+export type MenuEntry = {key: MenuKey; value: MenuItem};
+
+const MENU_ENTRY_CONFIG: Record<MenuKey, MenuItem> = {
+  board: {
+    localizationKey: "BoardSettings",
+    location: "board",
+    isModeratorOnly: true,
+    color: "backlog-blue",
+    icon: GeneralSettings,
+  },
+  participants: {
+    localizationKey: "Participants",
+    location: "participants",
+    isModeratorOnly: false,
+    color: "value-violet",
+    icon: Participants,
+  },
+  appearance: {
+    localizationKey: "Appearance",
+    location: "appearance",
+    isModeratorOnly: false,
+    color: "poker-purple",
+    icon: SettingsAppearance,
+  },
+  share: {
+    localizationKey: "ShareSession",
+    location: "share",
+    isModeratorOnly: false,
+    color: "planning-pink",
+    icon: Share,
+  },
+  export: {
+    localizationKey: "ExportBoard",
+    location: "export",
+    isModeratorOnly: false,
+    color: "backlog-blue",
+    icon: FileDefault,
+  },
+  feedback: {
+    localizationKey: "Feedback",
+    location: "feedback",
+    isModeratorOnly: false,
+    color: "value-violet",
+    icon: SettingsFeedback,
+  },
+  profile: {
+    localizationKey: "Profile",
+    location: "profile",
+    isModeratorOnly: false,
+    color: "poker-purple",
+    icon: "profile",
+  },
+};
+
+export const MENU_ENTRIES = Object.entries(MENU_ENTRY_CONFIG).map(([key, value]) => ({key, value}) as MenuEntry);
+
+// shorthand to enable all menu items
+export const ENABLE_ALL: Record<MenuKey, boolean> = {
+  board: true,
+  participants: true,
+  appearance: true,
+  share: true,
+  export: true,
+  feedback: true,
+  profile: true,
+};
```

Comment:
Suggestion: Add a `feedbackEnabled` const in this file, get the value from an environment variable and use the value of it inside the `ENABLE_ALL` record.

Reply:
Also, maybe a different name like `enabledFeedbackItems` would fit better

Reply:
The `feedbackEnabled` comes from the store which in turn retrieves it from the `ServerInfo`. 

The idea of the const `ENABLE_ALL` was to be able to use it as a shorthand to avoid having to enable all menu items manually, while still retaining the option to overwrite it.
```tsx
<SettingsDialog enabledMenuItems={{...ENABLE_ALL, feedback: feedbackEnabled}} />
```

Remember, the reason this refactor was required in the first place is to be able to dynamically change which menu items are available, and which aren't.